### PR TITLE
Milestone 1: readme.md

### PR DESCRIPTION
**Pull request by sanieni6**
This is the initial version of my project with changes that need a review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-# Hello-world
\ No newline at end of file
+# Hello-world 👏
```
Comment (brainconnect93) [inline README.md]: - [x] Kindly make your README more descriptive and professional by adding the table of content to it as seen in the template, 
  - Add the Future Feature section to it and include 2-3 aspects of the project that you will add in the future
  - Kindly add the MIT license section to it and be sure the Link works perfectly by taking us to the licensed copyright you add in your project root directory.

Here is the new README template given to us by MIcroverse: [LINK](https://github.com/microverseinc/readme-template)

-N.B: Kindly remove the Microverse Logo at the top of the README file, optionally you can add your logo if you have one

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @sanieni6, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x] I would kindly suggest you make your Pull Request title to be more descriptive to denote what the project you worked on is all about and also give it summary of what the project entails. Kindly work on it, you can use this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for a better understanding. 👍 👍

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sanieni6) [reply README.md]: hi @brainconnect93 , should i create a new pull request with all the changes?

**Comment by sanieni6** [comment]
## Milestone 1: readme.md
Changes done:
+ Change the title of the pull request
+ Add the MIT license
+ Make README more descriptive and professional

File: README.md (hunk 1 above)
Comment (AHP15) [reply README.md]: Hi @sanieni6, I'm your second reviewer: No You don't need a new pull request 👍

**Review by AHP15 — APPROVED** [review]
Hi @sanieni6, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Good Readme file
✅ Good commit message
✅ No linter errors
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**